trustme/__init__.py

```diff
@@ -316,18 +296,17 @@ def create_child_ca(self):
         path_length = self._path_length - 1
         return CA(parent_cert=self, path_length=path_length)
 
-    def issue_cert(self, *identities, **kwargs):
-        # type: (Text, Optional[Union[Text, datetime.datetime]]) -> LeafCert
-        # PY3: (str, Optional[str], Optional[str], Optional[str], Optional[datetime.datetime]) -> LeafCert
-        """issue_cert(*identities, common_name=None, organization_name=None, \
-        organization_unit_name=None, not_after=None)
-
-        Issues a certificate. The certificate can be used for either servers
+    def issue_cert(
+        self,
+        *identities: str,
+        common_name: Optional[str] = None,
```

Comment:
Can we keep the kwargs keyword only?

Reply:
Everything after an `*args` parameter is keyword only (unless I'm confused)

Reply:
Oh, right, that makes sense.